darts/models/forecasting/tcn_model.py

```diff
@@ -332,7 +333,11 @@ def __init__(self,
 
         kwargs['input_chunk_length'] = input_chunk_length
         kwargs['output_chunk_length'] = output_chunk_length
-
+        if(dropout_fn is None):
+            dropout_fn = TorchNativeDropout(0.0)
+        if isinstance(dropout_fn, McCompatibleDropout) and (likelihood is not None):
+            logger.warning("Enabling Likelihood and Monte Carlo Dropout makes model doubly stochastic, \
```

Comment:
Actually I don't think we need this warning - anyway fitting deep architectures to approximate other distributions using stochastic gradient descent with momentum heuristics to approximate negative log likelihoods is also a happy mess :)
IMO it is sufficient to describe precisely in the docstring what setting `enable_mc_dropout=True` in `predict()` does (with a reference to the paper), and maybe say that, if used along likelihoods, it will provide an additional source of stochasticity.